src/status_im/subs/root.cljs

```diff
@@ -140,7 +140,7 @@
 (reg-root-key-sub :communities/channels-permissions :community-channels-permissions)
 (reg-root-key-sub :communities/requests-to-join :communities/requests-to-join)
 (reg-root-key-sub :communities/community-id-input :communities/community-id-input)
-(reg-root-key-sub :communities/resolve-community-info :communities/resolve-community-info)
+(reg-root-key-sub :communities/fetching-community-info :communities/fetching-community-info)
```

Comment:
Since you're renaming the subscription key, I think `info` doesn't add any meaning to the name because everything is an info of sorts, so just `:communities/fetching-community` would do it.